membership-baseline.md

```diff
@@ -0,0 +1,169 @@
+
+## Engagement and Access
+We strive to keep our community inclusive and open to all individuals and organizations. Non-members are encouraged to participate in projects, attend meetings, join mailing lists, and take part in events. Most of our resources and activities are readily available to everyone, fostering a collaborative and accessible environment.
+
+## Membership Benefits
+While most resources are accessible to non-members, becoming a member of the null Community provides additional benefits such as enhanced involvement in core team activities, increased networking opportunities, and the ability to shape the community's direction through potential board membership. Membership also signifies a deeper commitment and dedication to the community's goals and values.
+
+# Membership Benefits:
+At null, we offer a range of exclusive benefits to our members, including:
+
+**Access to Cutting-Edge Security Resources:**
+
+- Online library of security research papers, articles and access to hundreds of events and their resources
+    - https://github.com/null-open-security-community/Documentation/tree/master/null-Research-Papers
+    - https://blog.null.community/
+    - https://null.community/archives
+- Professional mentoring programs
+- Priority access to null Study groups
+    - Cloud Security
+    - Crypto
+    - Web application security
+    - Container security(WIP)
+- Few other work in progress benefits:
+    - Access to the GitHub Student Pack
+    - Visual Studio Benefits, providing you with valuable resources for your security projects.
+    - One month free pluralsight membership
+    - Free access to security training programs by - <Partner with learning platforms/companies and ask for free training subscriptions>
+- <Add more benefits here> related to learning resources
+
+**Note:** We can not promise the work in progress benefits as it mainly depends on the vendor and their community partnership policy.
+
+# Networking Opportunities:
+
+- Engage with like-minded professionals through discussion groups such as Discord, Whatsapp and social media handles
+- Attend chapter wise security community meetups, workshops and various other events. More details at https://null.community/chapters.
+
+
+# Discounts and Offers:
+
+- Being part of null gives you an added advantage of getting special priority discounts on global and participating regional security conferences, training programs or any other events with which null has partnered with. Giving you another reason to connect with industry leaders and expand your professional network.
+    - Few previous partnership are mentioned below 
+        - Nullcon
+        - Rootconf
+        - BSides Delhi
+        - Threatcon
+        - BSides Bangalore
+# Individual Membership Benefits
```

Comment:
we should also make a section for student benefits. 

a useful benifit for students could be to get mentorship from community members who put time out for null mentorship.

Reply:
made a note of this